src/main/java/com/cairn/waypoint/dashboard/endpoints/homeworkquestion/SimplifiedHomeworkQuestionDto.java

```diff
@@ -0,0 +1,61 @@
+package com.cairn.waypoint.dashboard.endpoints.homeworkquestion;
+
+import com.cairn.waypoint.dashboard.entity.HomeworkQuestion;
+
+import java.time.LocalDateTime;
+
+public class SimplifiedHomeworkQuestionDto {
```

Comment:
Let's use the annotated approach as opposed to explicit getters/setters here. You can utilize `@Data` to generate these via Lombok